providers/go.mod

```diff
@@ -0,0 +1,21 @@
+module k8s.io/cloud-provider-gcp/providers
+
+go 1.15
+
+require (
```

Comment:
In the root go.mod we have a replace stanza to pin k8s.io modules to specific versions. Let's be consistent and do the same here.

Reply:
Updated